app/filters.php

```diff
@@ -36,3 +36,55 @@
 add_filter('get_search_form', function () {
     return view('forms.search');
 });
+
+/**
+ * Modify oEmbed URL parameters.
+ */
+add_filter('embed_oembed_html', function ($cache, $url, $attr, $post_id) {
+    preg_match('/src="([^"]*)"/i', $cache, $sources);
+    if (!empty($sources)) {
+        $src = $sources[1];
+    }
+    if (!empty($src) && !empty($url)) {
+        $is_youtube = strpos($src, 'youtube') !== false;
```

Comment:
is it always youtube when embedding? they have short url service youtu.be

Reply:
Good point but I tested it and verified that the oEmbed does resolve to the full URL before this check comes. So it works.

Reply:
Great!